themes/base/theme.css

```diff
@@ -50,7 +50,10 @@
 ----------------------------------*/
 .ui-state-default,
 .ui-widget-content .ui-state-default,
-.ui-widget-header .ui-state-default {
+.ui-widget-header .ui-state-default,
+.ui-button,
+html .ui-button.ui-state-disabled:hover,
+html .ui-button.ui-state-disabled:active {
```

Comment:
Comment yes, but not support i don't think, since its not about browser support so much as just needing greater specificity.